src/Modules/Events/AgencyEventListContainer.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import SpinnerComponent from '../General/SpinnerComponent';
+
+import axios from 'axios';
+import { API_URL } from '../../Utils/Urls';
+import { EventHandler } from '../../Utils/EventHandler';
+import EventListComponent from './EventListComponent';
+
+const AgencyEventListContainer = props => {
+  const { agencyId } = useParams();
+  const [loading, setLoading] = useState(false);
+  const [error, setErrors] = useState(false);
+  const [targetAgency, setTargetAgency] = useState({});
+  const [agencyEvents, setAgencyEvents] = useState([]);
+
```

Comment:
General question. Does any of this content depend upon the user having landed on the main page or search results? We are thinking an agency would like to this page, and there would not be any established states to tap into.

Reply:
Good question. We do need the `agencyId` in the params. For example: `http://root/agency/events/123`.

Other than that, the user can go directly to the link. The `useState` is component level only. Meaning redux is not needed.